docs/trace/customizing-the-sdk/README.md

```diff
@@ -142,7 +146,7 @@ it is built.
 using OpenTelemetry;
 using OpenTelemetry.Trace;
 
-using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+var tracerProvider = Sdk.CreateTracerProviderBuilder()
```

Comment:
Add `tracerProvider.Dispose` for these examples as well?

Reply:
do we need? they are not full code, but just snippets.

Reply:
> they are not full code, but just snippets.

Why remove `using` in the first place? 😄

I guess if we are removing `using` then we could add these three lines to snippets to stay consistent and encourage best practices.
```csharp
// ....
// Dispose at application shutdown
tracerProvider.Dispose()
```

Reply:
> Why remove using in the first place? 😄

Have seen a lot of users copying it as-is, and unintentionally disposing too early!.